osv/impact.py

```diff
@@ -120,8 +120,9 @@ def ranges(self):
 class RepoAnalyzer:
   """Repository analyzer."""
```

Comment:
There's quite a bit of nuance going on in `get_affected`, potentially antithetical to the constructor. I think it would be worthwhile for readability/comprehensibility to surface this either in the class docstring or the constructor's docstring

Reply:
done!